src/js/node/tty.ts

```diff
@@ -1,81 +1,74 @@
+// Hardcoded module "node:tty"
+
 // Note: please keep this module's loading constrants light, as some users
 // import it just to call `isatty`. In that case, `node:stream` is not needed.
+
 const {
   setRawMode: ttySetMode,
   isatty,
   getWindowSize: _getWindowSize,
 } = $cpp("ProcessBindingTTYWrap.cpp", "createBunTTYFunctions");
 
 const { validateInteger } = require("internal/validators");
+const fs = require("node:fs");
```

Comment:
This should be internal/fs/streams.ts since we only need ReadStream.

Reply:
and internal/fs/streams.ts should be modified to not require node:fs. and node:fs/promises should be simplified to have FileHandle as a separate export.